lib/database_cleaner.rb

```diff
@@ -11,6 +11,8 @@ class << self
       :app_root,
       :logger=,
       :logger,
+      :cleaners,
+      :cleaners=,
```

Comment:
@botandrose Isn't this a non-backwards compatible change? What happens if someone is using `DatabaseCleaner.cleaners` in their code? They might get a `NoMethodError`, right?

Reply:
@etagwerker Hmm, I'm not understanding what you mean? This PR _adds_ the `DatabaseCleaners.cleaners` method. It also deprecates `DatabaseCleaners.connections` but doesn't remove it yet, so it should be Semver forwards compatible. Can you explain more about your concern? Maybe I am misunderstanding something. :)

Reply:
@botandrose Oh, good point! I misread the set of changes. Sorry about that. 👍

Reply:
All good! Thank you for the review! 😊